op-mode-definitions/reset-dhcp-server-lease.xml.in

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="reset">
+    <children>
+      <node name="dhcp-server">
+        <properties>
+          <help>reset DHCP server lease</help>
+        </properties>
+        <children>
+          <node name="lease">
+            <properties>
+              <help>DHCP server lease</help>
+            </properties>
+            <children>
+              <tagNode name="ip">
```

Comment:
Why can't the `lease` CLI node be the tagNode? Saves one level?

Reply:
What if we’ll reset nodes by hw address?
Reset it just we don’t have “clear” in our CLI
If it is not issue I preferred also to use “clear” command
How should it be in CLI?
reset dhcp-server lease x.x.x.x

Reply:
Done
```
clear dhcp-server lease 100.64.0.18
```